CMP1903M A01 2223/Pack.cs

```diff
@@ -73,54 +74,54 @@ public static bool shuffleCardPack(int typeOfShuffle)
             }
             else
             {
-                return false;
+                return false; // if there is any integer passed through other than the three corresponding to the shuffles it will return false
             }
         }
         // This method will deal one card
         public static Card Deal()
         {
-            if (pack.Count == 0)
+            if (pack.Count == 0) // checks if the pack has been built
             {
-                Console.WriteLine("The pack is empty");
-                return null;
+                Console.WriteLine("The pack is empty"); // if the pack is empty will write to the console that the pack is empty
+                return null; // indicates nothing has been reurned and ends
             }
             else
             {
-                Card FirstCard = pack[0];
-                pack.RemoveAt(0);
-                return FirstCard;
+                Card FirstCard = pack[0]; // Defines Firstcard as the first item in the pack
+                pack.RemoveAt(0); // Will remove the first item in the pack
+                return FirstCard; // returns the first item in the pack
                 
 
             }
 
         }
-        // This method deals an amount of cards specified by the user
-        public static List<Card> DealCard(int amount)
+        // This method deals an amount of cards passed through
+        public static List<Card> DealCard(int amount) 
         {
-            if (pack.Count == 0)
+            if (pack.Count == 0) // checks if the pack has been built
             {
-                Console.WriteLine("The pack is empty");
-                return null;
+                Console.WriteLine("The pack is empty"); // if the pack is empty will write to the console that the pack is empty
+                return null; // indicates nothing has been reurned and ends
             }
             else
             {
-                int packPointer = 1;
-                while (packPointer <= amount)
+                int packPointer = 1; // defines the pack pointer as 1
+                while (packPointer <= amount) // while the pack pointer is less than or equal to the amount of cards needed to be returned
```

Comment:
output a messasge like "error amount of cards is more than the cards in the pack" when amount is higher than packpointer